GPU/GPUCommon.cpp

```diff
@@ -74,13 +69,6 @@ GPUCommon::GPUCommon(GraphicsContext *gfxCtx, Draw::DrawContext *draw) :
 
 	UpdateVsyncInterval(true);
 	ResetMatrices();
-
-	PPGeSetDrawContext(draw);
```

Comment:
Software rendering still uses this, though.  It's for font drawing.

-[Unknown]

Reply:
Which is why it's still called, but now from SoftGPU.

Though, I guess it could just remain here..